recipes/rbdl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "rbdl" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/{{ name }}/{{ name }}/get/v{{ version }}.tar.gz
+  sha256: a73bfe37b7a86d954fdb57cc88cdc8e08d35a6799745dad395da491daaad504f
+
+build:
+  number: 0
+  skip: True  # [win]
+  rpaths:
+    - lib/
+    - lib/rbdl/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - eigen
```

Comment:
Do you need to have eigen as a runtime dependency?

Reply:
Yep, is 'host' not the right one for that?

Reply:
nope, you need another "run" section below this one :wink: